src/components/VFilters/VFilterChecklist.vue

```diff
@@ -20,40 +20,55 @@
         <VLicense v-if="filterType === 'licenses'" :license="item.code" />
         <template v-else>{{ itemLabel(item) }}</template>
       </VCheckbox>
-      <button
-        v-if="filterType === 'licenses'"
-        :ref="`${item.code}licenseIcon`"
-        :aria-label="$t('browse-page.aria.license-explanation')"
-        class="text-dark-charcoal-70 appearance-none"
-        type="button"
-        @click.stop="toggleLicenseExplanationVisibility(item.code)"
-      >
-        <VIcon :icon-path="helpIcon" />
-      </button>
+
+      <!-- License explanation -->
+      <VPopover v-if="filterType === 'licenses'">
```

Comment:
Let's pass the `label` prop here with the value of the license definition. That way the popover label is read out when it is opened and we don't have to mess with moving the focus away from the default of the close button (which I think makes the most sense ergonomically to auto-focus when the popover is opened).

Reply:
I did not understand what you mean by license definition. A license can have several elements, each with their own definition (eg. BY-NC-SA with 3 elements & 3 definitions).

Reply:
Ah, you're right. We might need to find a compromise for screen reader folks to get that information quickly upon opening the popover. Maybe something to explore later on though :thinking: Otherwise the aria-label would be a huge wall of text for multi-element licenses.

Reply:
Yeah, we can separate this into a separate issue to at least move past the post-launch bugs milestone.